todo_app/tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,73 @@
+import os
+import pytest
+from time import sleep
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from dotenv import load_dotenv
+from todo_app import app
+from todo_app.data.trello_items import TrelloItems
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
```

Comment:
Nice work on this, it's rare these end-to-end tests just pass straight out the door on my machine but this was perfectly smooth 😄